src/asynciterable/_sleep.ts

```diff
@@ -18,6 +18,10 @@ export function sleep(dueTime: number, signal?: AbortSignal) {
       resolve();
     }, dueTime);
 
+    if (unref) {
+      id.unref();
```

Comment:
Is IxJS meant to run in browser environments? If so, then we will also need:

```suggestion
    if (unref && 'unref' in id) {
      id.unref();
```